src/main.cpp

```diff
@@ -100,6 +104,8 @@ int main(int argc, char *argv[])
     }
 
     std::vector<unsigned char> bytecode;
+    // TODO - Allow reading debug information from file.
+    DebugInfo *debug = nullptr;
```

Comment:
There's not actually any reason why this needs to be a pointer, is there? What was wrong with it being a local variable?